ViewControllers/MostPopularCollectionViewController.swift

```diff
@@ -91,16 +82,34 @@ class MostPopularViewController: UIViewController, UICollectionViewDelegate {
         button.translatesAutoresizingMaskIntoConstraints = false
         button.setTitle("See more", for: .normal)
         button.titleLabel?.font = UIFont.systemFont(ofSize: 14, weight: .semibold)
-        button.tintColor = UIColor(named: "acidGreen")
+        button.tintColor = UIColor(named: "titleColor")
         button.alpha = 0
         return button
     }()
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        title = "Most popular"
-        navigationController?.navigationBar.titleTextAttributes = [.foregroundColor: UIColor.white, .font: UIFont.systemFont(ofSize: 24, weight: .heavy)]
+        setupUI()
+        
+        movieDataService.getMostPopularMoviesList { [weak self] (moviesList: [Movie]?) in
```

Comment:
Just a bit of syntactic sugar, we don't need to specify the type in the closure and can leave to the swift's automatic type inference
```suggestion
        movieDataService.getMostPopularMoviesList { [weak self] moviesList in
```